mubench.pipeline/tasks/implementations/publish_findings.py

```diff
@@ -92,7 +91,7 @@ def __to_postable_data(self, run_info, result, postable_potential_hits: List[Dic
         data = self._to_markdown_dict(run_info)
         data["result"] = result
         data["potential_hits"] = postable_potential_hits
-        data["timestamp"] = self.run_timestamp
+        data["timestamp"] = run_info["timestamp"]
```

Comment:
We don't need to explicitly copy the timestamp. The entire `run_info` is copied into the `data` three lines above.